src/ontology/mondo-edit.obo

```diff
@@ -436528,10 +436528,8 @@ is_a: MONDO:0001713 {source="DOID:13636", source="MESH:D005199", source="Orphane
 is_a: MONDO:0003225 {source="PMID:31953710"} ! bone marrow disorder
 is_a: MONDO:0005570 ! hematologic disorder
 is_a: MONDO:0015161 {source="Orphanet:84"} ! multiple congenital anomalies/dysmorphic syndrome without intellectual disability
-is_a: MONDO:0018234 ! dysostosis
 is_a: MONDO:0019054 ! congenital limb malformation
 is_a: MONDO:0021190 {source="MESH:D005199", source="MONDO:Redundant", source="NCIT:C62505"} ! DNA repair disease
-is_a: MONDO:0100137 {source="PMID:29804726"} ! telomere syndrome
```

Comment:
@yshwetar we do not only remove the SubClassOf, we need to keep track of what SubClassOf we remove, including the original source and your orcid (to indicate you reviewed it)
Please see instructions here: https://mondo.readthedocs.io/en/latest/editors-guide/g-logical-axioms/#excluded-subclassof:~:text=non%2Dlogical)%20axiom.-,Excluded%20subclassOf,%C2%B6,-Rather%20than%20removing

Reply:
note the addition of your orcid as a source for the "excluded subclass of" is not in the documentation yet (it is a relatively new sop)